src/main/java/StringAddCal.java

```diff
@@ -0,0 +1,52 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCal {
+    public static int splitAndSum(String input) throws RuntimeException {
+        if (nullOrEmpty(input) == false) {
```

Comment:
요구사항에 따라 '빈 문자열 또는 null 값을 입력할 경우'에만 0을 반환해야 하기 때문에, 조건식이 참이 되어야 하지 않을까요?
```suggestion
        if (nullOrEmpty(input)) {
```

Reply:
수정 완료